src/notification-card/notification-card-internal.tsx

```diff
@@ -30,6 +56,10 @@ function NotificationCardInternal(
   }: NotificationCardProps,
   ref?: React.Ref<HTMLDivElement>,
 ) {
+  const highlightVariant = inverted
+    ? invertedDecorVariantMap[variant].highlightVariant
+    : decorVariantMap[variant].highlightVariant;
```

Comment:
As per the suggestion above:
```suggestion
    ? invertedDecorVariantMap[variant]
    : decorVariantMap[variant];
```